cmd/authority-claimer/src/claimer.rs

```diff
@@ -48,66 +46,76 @@ pub enum ClaimerError<
 /// `BrokerListener`, a `DuplicateChecker` and a `TransactionSender`.
 #[derive(Debug)]
 pub struct DefaultClaimer<
-    B: BrokerListener,
+    R: Repository,
     D: DuplicateChecker,
     T: TransactionSender,
 > {
-    broker_listener: B,
+    repository: R,
     duplicate_checker: D,
     transaction_sender: T,
 }
 
-impl<B: BrokerListener, D: DuplicateChecker, T: TransactionSender>
-    DefaultClaimer<B, D, T>
+impl<R: Repository, D: DuplicateChecker, T: TransactionSender>
+    DefaultClaimer<R, D, T>
 {
     pub fn new(
-        broker_listener: B,
+        repository: R,
         duplicate_checker: D,
         transaction_sender: T,
     ) -> Self {
         Self {
-            broker_listener,
+            repository,
             duplicate_checker,
             transaction_sender,
         }
     }
 }
 
 #[async_trait]
-impl<B, D, T> Claimer for DefaultClaimer<B, D, T>
+impl<R, D, T> Claimer for DefaultClaimer<R, D, T>
 where
-    B: BrokerListener + Send + Sync + 'static,
+    R: Repository + Send + Sync + 'static,
     D: DuplicateChecker + Send + Sync + 'static,
     T: TransactionSender + Send + 'static,
 {
-    type Error = ClaimerError<B, D, T>;
+    type Error = ClaimerError<R, D, T>;
 
     async fn start(mut self) -> Result<(), Self::Error> {
         trace!("Starting the authority claimer loop");
         loop {
-            let rollups_claim = self
-                .broker_listener
-                .listen()
-                .await
-                .context(BrokerListenerSnafu)?;
-            trace!("Got a claim from the broker: {:?}", rollups_claim);
+            let (rollups_claim, iconsensus) =
+                self.repository.get_claim().await.context(RepositorySnafu)?;
+            trace!("Got a claim from the repository: {:?}", rollups_claim);
```

Comment:
We should adjust log levels to match what's been done for 1.5.x:
- [chore: improve logging](https://github.com/cartesi/rollups-node/commit/7374587024e2657d12310ff96cb2a1f3c3849d6e)
- [chore(authority-claimer): adjust log levels](https://github.com/cartesi/rollups-node/commit/d15df1b91dcc9ded6547492279c59c9714d7125b)